src/metabase/db/liquibase.clj

```diff
@@ -198,18 +202,18 @@
     (try
       (force-release-locks! liquibase)
       (catch Exception e
-        (log/error e (trs "Unable to release the Liquibase lock after a migration failure"))))))
```

Comment:
This code was being run in cases sitatuions to migration failures already.

Reply:
Hmm, we probably want to remove this translation - but either way we'll conflict with the outstanding the [PR to fix all the logs ](https://github.com/metabase/metabase/pull/41067#issuecomment-2039120716), so not worth another revision now.